RussianFilter/sections/specific.txt

```diff
@@ -36,6 +36,8 @@ oka.fm##.rek-module
 pickpoint.ru##.bn-index-ad1
 mangalib.me##.ads
 gorlonos.com###DDWidgetSearch
+||porno-chiks.com/player/html.php?aid=pause*
```

Comment:
better add more specific rule
`||porno-chiks.com/player/html.php?aid=pause_html&video_id=*&*referer=`

Reply:
no, it is ok